### PR TITLE
[WIP] Add more recipes

### PR DESCRIPTION
**Pull request by ChrisWeissmann**
Added 2 recipies.
Not sure on the date brewed and how much water was used.
Also the wheat beer ingrediants are in %. If the gramm amount is found would be great to add.

### CODE REVIEW

**Comment by ChrisWeissmann** [comment]
Ref to  #7

File: _posts/2019-03-07-WinterAle.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+categories: recipes
+---
+
+# Nerdkiez Winter Ale
+
+First time brewed: 8 December 2018
+
+Brewed again 2 March 2019
+
+![Dark Winter Ale](/assets/winterale.jpg)
```
Comment (c7h) [inline _posts/2019-03-07-WinterAle.md]: ist das asset erreichbar?

File: index.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
  layout: default
+ title: Home
 ---
 
-Lorem ipsum for now?
\ No newline at end of file
+Lorum ipsum for now :)
```
Comment (c7h) [inline index.md]: Da mach ich bald mal was rein.

**Comment by c7h** [comment]
@ChrisWeissmann kannst du mal schauen warum travis 🤖  fehlschlaegt? 💥 

```
- ./_site/404.html
  *  internally linking to /about, which does not exist (line 49)
     <a href="/about" class="btn">About</a>
- ./_site/about.html
  *  internally linking to /about, which does not exist (line 49)
     <a href="/about" class="btn">About</a>
- ./_site/index.html
  *  internally linking to /about, which does not exist (line 49)
     <a href="/about" class="btn">About</a>
- ./_site/jekyll/update/2019/03/06/welcome-to-jekyll.html
  *  internally linking to /about, which does not exist (line 51)
     <a href="/about" class="btn">About</a>
- ./_site/recipes/2018/02/01/Wheat.html
  *  internally linking to /about, which does not exist (line 51)
     <a href="/about" class="btn">About</a>
- ./_site/recipes/2018/09/06/Helles.html
  *  internally linking to /about, which does not exist (line 51)
     <a href="/about" class="btn">About</a>
- ./_site/recipes/2019/03/01/WinterAle.html
  *  internally linking to /about, which does not exist (line 51)
     <a href="/about" class="btn">About</a>
htmlproofer 3.10.2 | Error:  HTML-Proofer found 7 failures!
The command "./script/cibuild" exited with 1.
```

**Pull request merged**